api/minitwit_sim_api.rb

```diff
@@ -83,7 +80,7 @@ class SimAPI < Roda
         end
 
         r.is String do |username|
-          username = URI.unescape(username)
+          username = CGI.unescape(username)
```

Comment:
Is this related to the robucop rule change?

Reply:
Yes, it was brought to my attention through Rubocop that URI is deprecated.